Bio/Sequencing/Ace.py

```diff
@@ -284,10 +284,10 @@ def __init__(self, line=None):
             self.uorc = header[5]
 
 
-def parse(handle):
+def parse(source):
```

Comment:
This is technically an API change, and of minimal benefit?

Reply:
With this PR, `parse` accepts both file handles and file paths. If the argument name is called `handle`, it suggests it can accept file handles only.